### DISCUSSION TITLE
Place to meet and chat

### DISCUSSION THREAD

**Discussion by martinheidegger** [xaviergonz/mobx-keystone#246] [General]
When starting with mobx-keystone I was looking for means to chat with people that already set it up before me. What about opening/joining a [gitter channel](https://gitter.im/xaviergonz/mobx-keystone/)?

**Comment by xaviergonz** [comment]
To be honest I tend to skip chats and prefer issues instead (they seem more structured to me and easier to manage), but if enough people think it is a good idea I don't see why not go for it :)

**Comment by martinheidegger** [comment]
Waiting for more people to :+1: this issue then ;)

**Comment by Amareis** [comment]
I'm thinks about moving from mst to keystone and place where some not-so-big questions about migration (like #68 ) can be answered will be pretty good for me.

**Comment by terrysahaidak** [comment]
I also would like to have a place where we can all chat. But I don't think Gitter is the right choice. I would suggest Slack instead since probably all of us are using it and also it will fit the size of this community for now for sure.

**Comment by akaguny** [comment]
at the first time chat is good.
for me spectrum looks good.
with gitter is so difficult to track action/answer and find success answers (seems like mobx gitter)
